### ISSUE TITLE
Failed build on missing Common.Logging and Quartz 

### ISSUE THREAD

**Issue by Sebastian-Roth** [FOGProject/fog-client#88]
Building the current master of fog-client fails with the following warnings and errors:
```
C:\Program Files\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3245: Could not resolve this reference.
Could not locate the assembly "Common.Logging, Version=3.3.1.0, Culture=neutral, ...

C:\Program Files\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3245: Could not resolve this reference.
Could not locate the assembly "Common.Logging.Core, Version=3.3.1.0, Culture=neutral, ...

C:\Program Files\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3245: Could not resolve this reference.
Could not locate the assembly "Quartz, Version=2.4.1.0, Culture=neutral, ...

PowerManagement\PowerManagement.cs(22,7): error CS0246:
The type or namespace name 'Quartz' could not be found (are you missing a using directive or an assembly reference?) ...
...
```

**Comment by Sebastian-Roth** [comment]
Ah, I see, packet versions need updating in [.\Modules\Modules.csproj](https://github.com/FOGProject/fog-client/blob/master/Modules/Modules.csproj#L93).

**Comment by Sebastian-Roth** [comment]
Should be auto-closed by this pull request #89 soon.

**Comment by Sebastian-Roth** [comment]
done
